### PR TITLE
clean parameters in Python

### PR DESCRIPTION
**Pull request by QiJune**
Please review #1749 first.

### CODE REVIEW

File: elasticdl/python/worker/worker.py [hunk 1]
```diff
@@ -468,14 +470,38 @@ def report_gradient_to_ps(self, grads):
...
-                ps_grads[ps_id].append((g, v.name))
+                if v.name not in ps_grads[ps_id]:
+                    ps_grads[ps_id][v.name] = g
+                else:
+                    if isinstance(g, tf.IndexedSlices):
+                        ps_grads[ps_id][v.name] = merge_indexed_slices(
+                            ps_grads[ps_id][v.name], g
+                        )
+                    else:
+                        ps_grads[ps_id][v.name] += g
```
Comment (workingloong) [inline elasticdl/python/worker/worker.py]: Why  `ps_grads[ps_id][v.name] += g` not `ps_grads[ps_id][v.name] = g` ?

Comment (QiJune) [reply elasticdl/python/worker/worker.py]: Please review #1749 first.

```python
                if v.name not in ps_grads[ps_id]:
                    ps_grads[ps_id][v.name] = g
                else:
                    if isinstance(g, tf.IndexedSlices):
                        ps_grads[ps_id][v.name] = merge_indexed_slices(
                            ps_grads[ps_id][v.name], g
                        )
                    else:
                        ps_grads[ps_id][v.name] += g
```

If the gradient comes for the first time, we `ps_grads[ps_id][v.name] = g`.
If the gradient with a same name for the second time, we will merge the gradient to the existing gradient.

For `IndexedSlices` type gradient, we merge them;
For Dense type gradient, we add them

Comment (workingloong) [reply elasticdl/python/worker/worker.py]: Got it!

**Comment by workingloong** [comment]
I found that line 470 to line 500  in ` elasticdl/python/worker/worker.py` are the same as  #1749

**Comment by QiJune** [comment]
@workingloong  Since I merge #1749 to this PR. So it's better to review #1749 first.

**Review by workingloong — APPROVED** [review]

**Pull request merged**